grassroots-backend/src/contacts/Contacts.controller.ts

```diff
@@ -3,9 +3,9 @@ import { ContactsService } from "./Contacts.service";
 import { ContactEntity } from "./entities/Contact.entity";
 import {
   ContactDTO,
-  CreateBulkContactRequestDto,
+  CreateBulkContactRequestDTO,
```

Comment:
I gotta stop fixing little things in the middle of big patches... Sorry about this!